vaadin-dev-server/src/main/java/com/vaadin/base/devserver/OpenInCurrentIde.java

```diff
@@ -164,7 +164,10 @@ static Optional<Info> findIdeCommand(List<Info> processes) {
     }
 
     private static String getCommandAndArguments(Info info) {
-        return info.commandLine().get();
+        if (info.commandLine().isPresent()) {
+            return info.commandLine().get();
+        }
+        return null;
```

Comment:
It can perhaps be simplified as
```suggestion
        return info.commandLine().orElse(null);
```

Reply:
Thanks, done